### DISCUSSION TITLE
What are the limitations?

### DISCUSSION THREAD

**Discussion by Temepest74** [dompdf/dompdf#3411] [Q&A] [answered]
I want to export any html to pdf. Can I do it or are there any limitations as if, idk, some css wont work or something like that?

**Comment by bsweeney** [comment, marked as answer]
There are some features of CSS that aren't yet supported. You can find the list of supported CSS on the [CSS Compatibility](https://github.com/dompdf/dompdf/wiki/CSSCompatibility) wiki page. One of the most requested features that is not yet supported is flexbox support.

Beyond that, probably the biggest limitation right now is around paging of some elements. First, Dompdf currently requires that a table cell fit on a single page, otherwise the entire row will move to the next page. Second, non-flow elements like floats are fully rendered before following elements, meaning a floating element larger than a page will move all following content to the next page.

Beyond that, though, it would help to know what you want to render. There's a [debug helper](https://eclecticgeek.com/dompdf/debug.php) you can use to test Dompdf's functionality.

**Reply by Temepest74** [reply]
Sadly, it wont do the job for me, but thank you

**Reply by bsweeney** [reply]
Sorry to hear that. Good luck finding the right project.
